bin/conjur-cli.rb

```diff
@@ -195,15 +189,15 @@ def test_select
 desc "Manage roles"
 command :role do |cgrp|
   cgrp.desc "Retrieve a role's API key"
+  cgrp.arg :role_id, :multiple
   cgrp.command :"retrieve-key" do |c|
     c.action do |global_options,options,args|
-      exit_now! "Must specify role ID" unless args.count == 1
-      
-      role_id = args.shift
-      
+
       connect
-      
-      exec "rake role:retrieve-key[#{role_id}]"
+
+      args.each do |id|
+        print `rake role:retrieve-key[#{id}]`
```

Comment:
Why the change to `print`?

Reply:
`exec "rake"` replaces the Ruby process with the rake process, while ``print `rake` `` launches a subprocess. In order to allow the loop to finish, I made the change.